### PR TITLE
Clear db during testing

### PR DESCRIPTION
**Pull request by sankaest**
| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️/❌
| New feature?  | ✔️/❌
| Breaks BC?    | ✔️/❌
| Fixed issues  | #458

### CODE REVIEW

File: src/Command/Fixture/SchemaClearCommand.php [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+                ->getRepository(Tag::class)
+                ->findAll())+
+            count($this->promise
+                ->getORM()
+                ->getRepository(User::class)
+                ->findAll())+
+            count($this->promise
+                ->getORM()
+                ->getRepository(Comment::class)
+                ->findAll());
```
Comment (BoShurik) [inline src/Command/Fixture/SchemaClearCommand.php:91]: findAll + count looks like huge overhead. Maybe replace it with count query?

File: src/Command/Fixture/SchemaClearCommand.php [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+            ->delete('user')
+            ->run();
+
+        $this->promise
+            ->getDatabaseProvider()
+            ->database()
+            ->delete('comment')
+            ->run();
+
+        return count($this->promise
```
Comment (BoShurik) [inline src/Command/Fixture/SchemaClearCommand.php:72]: Command must return code. 0 - success, 1 - failure etc (see https://www.redhat.com/sysadmin/exit-codes-demystified)

**Review by BoShurik — CHANGES_REQUESTED** [review]

File: src/Command/Fixture/SchemaClearCommand.php (hunk 1 above)
Comment (sankaest) [reply src/Command/Fixture/SchemaClearCommand.php:91]: I was looking for that, but could not find it (I am new to Yii3). Will try to find it.

File: src/Command/Fixture/SchemaClearCommand.php (hunk 2 above)
Comment (sankaest) [reply src/Command/Fixture/SchemaClearCommand.php]: Thank you for info!

**Review by samdark — APPROVED** [review]

**Comment by samdark** [comment]
Thanks.

**Pull request merged**